static/css/main.css

```
@@ -1087,6 +1091,11 @@ video {
   border-color: rgb(184 26 86 / var(--tw-border-opacity, 1));
 }
 
+.border-white-dark {
+  --tw-border-opacity: 1;
+  border-color: rgb(229 228 215 / var(--tw-border-opacity, 1));
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)